src-tauri/src/deeplink.rs

```diff
@@ -0,0 +1,46 @@
+use tauri::{Emitter, Url};
+
+use crate::APP_HANDLE;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone)]
+pub struct InstallMCPServerArgs<'a> {
+    query: &'a str,
```

Comment:
I think this could just be `String`, unless we need to explicitly borrow the query from the `Url` below?